modules/static/prometheus/client-model/initref

```diff
@@ -1 +1 @@
-147c58e9608a4f9628b53b6cc863325ca746f63a
+v0.6.1
```

Comment:
This is the latest release: https://github.com/prometheus/client_model/releases

The closest release to the original initref commit would have been:

```
➜  client_model git:(147c58e) git describe --all 147c58e9608a4f9628b53b6cc863325ca746f63a
tags/v0.2.0-6-g147c58e
```